app/code/Ludmila/LSNewClasses/Model/ShowTypes.phtml

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace Ludmila\LSNewClasses\Model;
+
+class showTypes
```

Comment:
The class name should be the same, as filename, so please rename showTypes to ShowTypes